src/components/TableBody.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[],
+};
+
+const PersonRow: React.FC<Props> = ({ people }) => (
+  <>
+    {people.map(person => {
+      const {
+        slug,
+        name,
+        sex,
+        born,
+        died,
+        motherName,
+        fatherName,
+      } = person;
```

Comment:
This is not the logic that you should keep in `PersonRow`, you should map your array outside and return a `PersonRow` for EACH person